shared/api.ts

```diff
@@ -17,6 +17,10 @@ import {
 import {TelemetryConstants} from './telemetry/TelemetryConstants';
 
 export const API_URLS: ApiUrls = {
+  trustedVerifiersList: {
+    method: 'GET',
+    buildURL: (): `/${string}` => '/v1/mimoto/trustedVerifiers',
```

Comment:
I don't see API in mimoto

Reply:
Yes swati it is not added yet we will add it after verifier team created and exposed the response uri endpoint. I have added the above code during the development thinking verifier team will complete generating endpoint before we raising pr for UI but it is not happened so i will remove the above code to avoid confusion.